provider/telegram/telegram.go

```diff
@@ -0,0 +1,42 @@
+package telegram
+
+import (
+	"strconv"
+
+	"github.com/messagebird/sachet"
+	"gopkg.in/telegram-bot-api.v4"
```

Comment:
the readme sez: "Use github.com/go-telegram-bot-api/telegram-bot-api for the latest version, or use gopkg.in/telegram-bot-api.v4 for the stable build."

I think "stable" is enough for sachet.